src/main/java/uk/gov/hmcts/reform/sscs/service/PanelCompositionService.java

```diff
@@ -23,22 +23,24 @@ public PanelCompositionService(CcdService ccdService, IdamService idamService) {
     }
 
     public void processCaseState(Callback<SscsCaseData> callback, SscsCaseData caseData, EventType eventType) {
-        if (caseData.getIsFqpmRequired() == null
-            || hasDueDateSetAndOtherPartyWithoutHearingOption(caseData)) {
-            ccdService.updateCase(caseData, callback.getCaseDetails().getId(),
-                EventType.NOT_LISTABLE.getCcdType(),
-                "Not listable",
-                "Update to Not Listable as the case is either awaiting hearing enquiry form or for FQPM to be set",
-                idamService.getIdamTokens());
-        } else {
-            if (eventType.equals(EventType.UPDATE_OTHER_PARTY_DATA)) {
-                caseData.setDirectionDueDate(null);
+        if (!(caseData.getState() != null && caseData.getState().equals(State.DORMANT_APPEAL_STATE))) {
```

Comment:
I think this can be simplified. As there is no specific path for if the state is null, you don't need that as part of this if condition. The condition is reasonably tricky to understand by reading it due to double negative and &&.
```suggestion
        if (caseData.getState() != State.DORMANT_APPEAL_STATE) {
```